_data/docs.yml

```
@@ -164,14 +164,14 @@ libs:
   libcudf:
     name: libcudf
     path: libcudf
-    desc: 'libcudf is a C/C++ CUDA library for implementing standard dataframe operations.'
+    desc: 'libcudf is a C/C++ CUDA library for implementing standard dataframe operations. Note that as of RAPIDS 24.02 libcudf docs are integrated into the Sphinx documentation alongside the cudf Python docs.'
```

```suggestion
    desc: 'libcudf is a C/C++ CUDA library for implementing standard dataframe operations. Note that as of RAPIDS 24.02 libcudf docs are integrated into the Sphinx documentation alongside the cuDF Python docs.'
```
Minor: changing capitalization for consistency with above.

If you feel like taking a min, you could consider adding a `\n` or `<br>` to see if either render okay; I do think it would be preferable to make it more visually obvious if it's possible.